pages/index.tsx

```diff
@@ -89,6 +104,71 @@ const Frontpage: NextPage<Props> = ({ featureFlags = { USE_IMPROVED_SEARCH: fals
             </HeroContent>
           </Hero>
         </Section>
+
+        <Section highlight>
+          <Heading level="h3" size="xl" style={{ marginBottom: "1.5rem" }}>
+            <FormattedMessage id="frontpage.relevantdatainfo.header" />
+          </Heading>
+
+          <GridContainer>
+            <StyledLink href={{ pathname: "https://statoilsrm.sharepoint.com/sites/Datatomany" }} target="_blank">
+              <CardGrid elevation="raised">
```

Comment:
I think this code should be refactored to a component because it's too repetitive